internal/action/command.go

```diff
@@ -634,16 +634,10 @@ func (h *BufPane) ResetCmd(args []string) {
 	}
 
 	option := args[0]
+	defaults := config.DefaultAllSettings()
 
-	defaultGlobals := config.DefaultGlobalSettings()
-	defaultLocals := config.DefaultCommonSettings()
-
-	if _, ok := defaultGlobals[option]; ok {
-		SetGlobalOptionNative(option, defaultGlobals[option])
-		return
-	}
-	if _, ok := defaultLocals[option]; ok {
-		h.Buf.SetOptionNative(option, defaultLocals[option])
+	if _, ok := defaults[option]; ok {
+		SetGlobalOptionNative(option, defaults[option])
```

Comment:
Still not checked for errors.
`reset ruler aaa` -> silently does nothing.

Reply:
Was this ever in scope of `reset`?

> `reset 'option'`: resets the given option to its default value

What you tried is reaching the same with `reset` for which `set(local)` is used.

Reply:
Ah indeed, sorry for the noise.

Reply:
Don't worry.
I never expect this topic explode in front of me.